bin/action_runner.py

```diff
@@ -21,18 +25,19 @@
 
 class StatusFile(object):
     """Manage a status file."""
+
     def __init__(self, filename):
         self.filename = filename
 
     def write(self, command, proc):
         with opener(self.filename, 'w') as fh:
-            yaml.dump(self.get_content(command, proc), fh)
+            yaml.safe_dump(self.get_content(command, proc), fh)
```

Comment:
Yay safe!

Reply:
fwiw this is going to use the slow pure-python dumper (very very slow)

The pattern I tend to use (or some variation thereof):

```python
import yaml

# could also *force* the c dumper by accessing it directly, though yaml can install without it if `libyaml-dev` isn't present
SafeDumper = getattr(yaml, 'CSafeDumper', yaml.SafeDumper)

yaml.dump(..., Dumper=SafeDumper)
```